src/main/java/Duke.java

```diff
@@ -1,10 +1,118 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
+    private static ArrayList<Task> tasksList = new ArrayList<>();
```

Comment:
Plural form should be used on names representing a collection of objects. For example: tasks